tests/integration/blockchain/test_blockchain.py

```diff
@@ -1338,6 +1381,39 @@ async def test_create_and_abandon_in_same_block(self):
             accepted=[]
         )
 
+    async def test_content_filtering(self):
+        user_chan = await self.get_claim(
+            await self.create_claim(claim_id_startswith='ab', is_channel=True, name="@some_channel"))
+        await self.create_claim(claim_id_startswith='cd', sign=user_chan, name="good_content")
+        bad_content = await self.get_claim(
+            await self.create_claim(claim_id_startswith='ef', sign=user_chan, name="bad_content"))
+        moderator_chan = await self.get_claim(
+            await self.create_claim(claim_id_startswith='ab', is_channel=True, name="@filters"))
```

Comment:
`claim_id_startswith` is pretty expensive operation (it generates lots of claims until the hash matches requested prefix), unless you actually need it, it's best not to use it.

Reply:
pushed, good find